test/e2e/upgradetest/upgrade_test.go

```diff
@@ -72,3 +72,67 @@ func TestResize(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// TestBackupForOldCluster tests that new backup sidecar could make backup from old cluster.
```

Comment:
we also test the cluster can recover from the newly made backup in this test, right?